composer.json

```diff
@@ -28,21 +28,16 @@
         }
     ],
     "require": {
-        "php": "^5.6 | ^7",
+        "php": "^7 || ^8",
         "behat/mink": "~1.7@dev",
         "php-webdriver/webdriver": "^1.4"
     },
     "require-dev": {
-        "mink/driver-testsuite": "dev-master"
```

Comment:
The phpunit config explicitly includes config to use this package - why are we removing it?

Reply:
I removed the phpunit config file.  It was extremely strange and only ran unit tests in a different module

The silverstripe module has no unit tests to run https://github.com/silverstripe/MinkFacebookWebDriver